homeassistant/components/knx/trigger.py

```diff
@@ -31,20 +31,15 @@
 CONF_KNX_INCOMING: Final = "incoming"
 CONF_KNX_OUTGOING: Final = "outgoing"
 
-TELEGRAM_TRIGGER_OPTIONS: Final = {
+
+TELEGRAM_TRIGGER_SCHEMA: Final = {
+    vol.Optional(CONF_KNX_DESTINATION): vol.All(cv.ensure_list, [ga_validator]),
     vol.Optional(CONF_KNX_GROUP_VALUE_WRITE, default=True): cv.boolean,
     vol.Optional(CONF_KNX_GROUP_VALUE_RESPONSE, default=True): cv.boolean,
     vol.Optional(CONF_KNX_GROUP_VALUE_READ, default=True): cv.boolean,
     vol.Optional(CONF_KNX_INCOMING, default=True): cv.boolean,
     vol.Optional(CONF_KNX_OUTGOING, default=True): cv.boolean,
 }
-TELEGRAM_TRIGGER_SCHEMA: Final = {
-    vol.Optional(CONF_KNX_DESTINATION): vol.All(
-        cv.ensure_list,
-        [ga_validator],
-    ),
-    **TELEGRAM_TRIGGER_OPTIONS,
```

Comment:
This was split up so the options can be used as `extra_fields` in device_trigger.py directly. Now we use selectors so we can merge the Schema here again. 

I didn't switch to `BooleanSelector` here since it just coerces to `bool` instead of checking values like `cv.boolean` does - eg. not raising for typos like `outgoing: "false"`, but assigning `true` since it is a truthy string.